### DISCUSSION TITLE
How many countries make up the EU today?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#790] [Galaxy Trivia] [answered]
**Bonus:** List the flags of the founding countries.

**Comment by SANKARAMDAS** [comment]
27 countries
The European Union ( EU ) is an economic and political union of 27 countries.

EU founding countries - 
🇩🇪 -Germany, to be more specific West Germany
🇫🇷 - France
🇮🇹 - Italy
🇧🇪 - Belgium
🇱🇺 - Luxembourg
🇳🇱 - The Netherlands

**Comment by peterbaccaro** [comment, marked as answer]
The European Union (EU) is an economic and political union of **27 countries**.

The organisation founded in 1957 which is now known as the European Union, originally had six members: 
- 🇧🇪 - Belgium
- 🇫🇷 - France
- 🇩🇪 - Germany
- 🇮🇹 - Italy
- 🇱🇺 - Luxembourg
- 🇳🇱 - Netherlands.
